src/backend/joanie/core/api/admin.py

```diff
@@ -438,6 +438,15 @@ class OrderViewSet(
     )
     ordering = "created_on"
 
+    @action(detail=True, methods=["DELETE"])
```

Comment:
No you have to use the `DestroyModelMixin` and override the `destroy` method. DRF always have a mixin for basic CRUD.

Reply:
According to your suggestion, I have added `DestroyModelMixin` into the `OrderViewSet`. 
I'll be careful about those type of actions from now on. Thanks again for the reminde :ok_hand: